governance-and-automation/automated-workflows/orphaned-resources-admin.md

```diff
@@ -24,4 +26,13 @@ The **Edit Policy** dialog opens where you can:
 ![Edit Policy Dialog](../../.gitbook/assets/set-up-automated-workflows-orphaned-dialog.png)
 
 
+{% hint style="info" %}
+
+**Please note:** The Orphaned Workspaces policy vulnerability is detected when there are 0 active owners assigned to a workspace. If there are less than 2 minimum owners assigned to a workspace, the [Minimum Number of Owners policy](../automated-workflows/minimum-number-of-owners-admin.md) vulnerability is detected. 
```

Comment:
umjesto less than 2 bi svuda stavio 1. ako je <2, a nije 0, ostaje samo 1, pa mislim da je bolje to tako i nazvati